.github/workflows/check.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       additional-env-vars: |
         NOT_CRAN=true
         QUARTO_PROFILE=development
+        TLG_CATALOG_PKG_BUILD_RENDER=TRUE
```

Comment:
@cicdguy 
if we are going with this approach - we would have to add that flag to our integration tests

Reply:
Hey @cicdguy. Ping on this. I got this merged and we need updates in our integration test pipelines.